src/game/mod.rs

```diff
@@ -2,11 +2,17 @@
 
 use bevy::prelude::*;
 
+mod animation;
 pub mod assets;
 pub mod audio;
 mod movement;
 pub mod spawn;
 
 pub(super) fn plugin(app: &mut App) {
-    app.add_plugins((audio::plugin, movement::plugin, spawn::plugin));
+    app.add_plugins((
+        audio::plugin,
+        movement::plugin,
+        animation::plugin,
```

Comment:
```suggestion
        animation::plugin,
        audio::plugin,
        movement::plugin,
```
Alphabetizing barring a reason not to :P